packages/i18n/src/locales/en/options/simone.yml

```diff
@@ -1,3 +1,7 @@
+bustAlignedButtons:
```

Comment:
If we move this to `simon.yml` the tests should pass (and it will get inherited by simone anyway)

Reply:
I moved the option translation from Simone to Simon.